src/GraphQL.Tests/Introspection/IntrospectionResult.cs

```diff
@@ -3,5 +3,23 @@
+
+        public static readonly string InputObjectBugQuery = @"
```

Comment:
I would keep these smaller ones in the original test file.  This massive one was put in a different file because of massiveness.